SPECS/azurelinux-rpm-macros/azurelinux-rpm-macros.spec

```diff
@@ -6,7 +6,7 @@
 %global __brp_python_bytecompile %{nil}
 Summary:        Azure Linux specific rpm macro files
 Name:           azurelinux-rpm-macros
-Version:        %{azl}.0
+Version:        3.0
 Release:        1%{?dist}
```

Comment:
From the size of this change, it might be worth to bump `Release` and add a note in the changelog.

Reply:
Good idea. There actually was a bump to version 2.0-27 from Daniel, who also helped with some of these changes. But now that we're on version 3.0, I'll bump it again.